pkg/controller/metrics/exporters/lvm_exporter/lvm_exporter.go

```diff
@@ -28,45 +30,77 @@ import (
 //struct for lvm  collector that contains pointers
 //to prometheus descriptors for each metric we expose.
 type lvmCollector struct {
-	mu         sync.Mutex
+	mu sync.Mutex
 	//volume metrics
-	IOPS            *prometheus.Desc
-	ReadThroughput  *prometheus.Desc
-	WriteThroughput *prometheus.Desc
-	ResponseTime    *prometheus.Desc
-	ServiceTime     *prometheus.Desc
-	Utilization     *prometheus.Desc
+	VolumeIOPS            *prometheus.Desc
+	VolumeReadThroughput  *prometheus.Desc
+	VolumeWriteThroughput *prometheus.Desc
+	VolumeResponseTime    *prometheus.Desc
+	VolumeServiceTime     *prometheus.Desc
+	VolumeUtilization     *prometheus.Desc
+	//Disk metrics
+	DiskIOPS            *prometheus.Desc
+	DiskReadThroughput  *prometheus.Desc
+	DiskWriteThroughput *prometheus.Desc
+	DiskResponseTime    *prometheus.Desc
+	DiskServiceTime     *prometheus.Desc
+	DiskUtilization     *prometheus.Desc
 }
 
 /* rr */
 //constructor for lvm collector that
 //initializes every descriptor and returns a pointer to the collector
 func newLvmCollector() *lvmCollector {
-	var volumeLabel = []string{"volume"}
+	var labelKeys = []string{"device"}
+
 	return &lvmCollector{
-		IOPS: prometheus.NewDesc("OpensSDS_Volume_IOPS_tps",
+		VolumeIOPS: prometheus.NewDesc("OpensSDS_Volume_IOPS_tps",
 			"Shows IOPS",
-			volumeLabel, nil,
+			labelKeys, nil,
 		),
-		ReadThroughput: prometheus.NewDesc("OpensSDS_Volume_ReadThroughput_KBs",
+		VolumeReadThroughput: prometheus.NewDesc("OpensSDS_Volume_ReadThroughput_KBs",
 			"Shows ReadThroughput",
-			volumeLabel, nil,
+			labelKeys, nil,
 		),
-		WriteThroughput: prometheus.NewDesc("OpensSDS_Volume_WriteThroughput_KBs",
+		VolumeWriteThroughput: prometheus.NewDesc("OpensSDS_Volume_WriteThroughput_KBs",
 			"Shows ReadThroughput",
-			volumeLabel, nil,
+			labelKeys, nil,
 		),
-		ResponseTime: prometheus.NewDesc("OpensSDS_Volume_ResponseTime_ms",
+		VolumeResponseTime: prometheus.NewDesc("OpensSDS_Volume_ResponseTime_ms",
 			"Shows ReadThroughput",
-			volumeLabel, nil,
+			labelKeys, nil,
 		),
-		ServiceTime: prometheus.NewDesc("OpensSDS_Volume_ServiceTime_ms",
+		VolumeServiceTime: prometheus.NewDesc("OpensSDS_Volume_ServiceTime_ms",
 			"Shows ServiceTime",
-			volumeLabel, nil,
+			labelKeys, nil,
 		),
-		Utilization: prometheus.NewDesc("OpensSDS_Volume_Utilization_prcnt",
+		VolumeUtilization: prometheus.NewDesc("OpensSDS_Volume_Utilization_prcnt",
 			"Shows Utilization in percentage",
-			volumeLabel, nil,
+			labelKeys, nil,
+		),
+		DiskIOPS: prometheus.NewDesc("OpensSDS_Volume_IOPS_tps",
```

Comment:
What do you mean here `OpensSDS`?

Reply:
@leonwanghui , Earlier we thought of naming with OpenSDS to identify metrics collected using opensds modules, but prometheus has naming stranded of using exporter identity to start the metric name, hence using 'lvm' instead if 'OpenSDS'.